docs/_specs/event-manager/event-manager.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: default
+---
+# Event manager
+
+Event manager handles events traffic within Dispatch. It connects events to functions using subscriptions,
+as well as allows creating event drivers, which serve as event sources.
+
+## Problem statement
+
+Functions are commonly associated with events and are executed in response to them. To effectively manage their
+relation, we need a way to handle events lifecycle (CRUD) as well as subscriptions and triggering.
+
+### User stories (examples)
+
+1. As a **developer**, I want to trigger an **event** by providing its **name/topic** and JSON-formatted **payload**, which will
+   in turn cause execution of one or more functions **subscribed** to that event.
+2. As a **developer**, I want to create a **subscription** between **event** and **function**. When event is triggered,
```

Comment:
between _an_ \*\*event\*\* and _a_ \*\*function\*\*. When _an_ event is triggered,